### ISSUE TITLE
Improve the "Add title..." and "Write" placeholders color contrast ratio

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#1302]
The new "Write placeholders" have an insufficient color contrast ratio. To be compliant with the WCAG at level AA, text should have a contrast ratio of at least 4.5:1 with its background.

<img width="520" alt="screen shot 2017-06-20 at 14 20 37" src="https://user-images.githubusercontent.com/1682452/27333290-aaf7f6bc-55c5-11e7-86bb-f83d955df17f.png">

<img width="538" alt="screen shot 2017-06-20 at 14 20 54" src="https://user-images.githubusercontent.com/1682452/27333291-aafbff50-55c5-11e7-8a6f-16922e933901.png">

If I'm not wrong, the current color is #ccd0d4 and gives a result of:
Luminosity Contrast Ratio: 1.55:1
https://jdlsn.com/color/?type=hex&color=ccd0d4&color2=ffffff

The only exception is when the text is bigger than approximately 24px or bigger than approximately 18.5px _and_ bold: in this case, the required contrast ratio threshold lowers to 3:1. 

The "Add title..." placeholder has a (computed) font-size of 32px but the color contrast ratio is just 2.93:1 and should be at least 3:1.

Note on colors:
I guess colors should be picked up from the new ones designed by @hugobaeta, https://codepen.io/hugobaeta/pen/grJjVp not sure `#ccd0d4` is there. I see the implementation sometimes uses a CSS generated content and sometimes a placeholder attribute, that's perfectly fine but worth nothing the current styling for the placeholder attribute is overriding the rules provided by core:

<img width="371" alt="screen shot 2017-06-20 at 14 32 51" src="https://user-images.githubusercontent.com/1682452/27333914-d0e508a4-55c7-11e7-82a3-623bd0e50f50.png">

they were introduced in https://core.trac.wordpress.org/changeset/36619 for good reasons. Maybe worth considering updating the core rules with a new (and accessible) shade of gray.

**Comment by jasmussen** [comment]
Nice. Looking at https://github.com/WordPress/gutenberg/blob/master/editor/assets/stylesheets/_variables.scss, what would be the lightest Hugo gray that would pass the test?

**Comment by afercia** [comment]
Against a white background, I think it's `#6c7781` which gives a 4.57:1 contrast ratio:
https://jdlsn.com/color/?type=hex&color=6c7781&color2=ffffff

**Comment by jasmussen** [comment]
I could accept that, but wouldn't #7e8993 work also? https://www.joedolson.com/tools/color-contrast.php?type=hex&color=%237e8993&color2=%23ffffff&alpha=1 — that gives a > 3:1 contrast ratio, which you suggest should be the least. 

I suggest this lighter color not for the visuals, but rather, I'd like for it to truly look like _placeholder text_, and not be confused with _actual text_, and the lighter we can make it, the better for that. If #7e8993 passes, even if not with flying colors (pardon the pun), I'd vote for that.

**Comment by afercia** [comment]
A 3:1 contrast ratio conforms to the WCAG level AA only for text bigger than 24px or 18.5px bold. So, #7e8993 is OK for the big "Add title..." but not sufficient for the smaller "Write your story/Write".

**Comment by jasmussen** [comment]
Ah understood.
